src/test/java/itest/GraphQLTest.java

```diff
@@ -18,9 +18,13 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.hasItem;
 import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
```

Comment:
Could you replace these start-imports with explicit imports for whichever classes and static methods are used?